gensrc/proto/internal_service.proto

```diff
@@ -133,7 +133,7 @@ message PTabletWriterOpenRequest {
     required bool need_gen_rollup = 7; // Deprecated
     optional int64 load_mem_limit = 8;
     optional int64 load_channel_timeout_s = 9;
-    optional bool is_vectorized = 20;
+    optional bool DEPRECATED_is_vectorized = 20;
```

Comment:
You can not change the variable name. Instead, add a DEPRECATED flag.
```
optional bool is_vectorized = 20; // DEPRECATED
```

Reply:
This variable is no longer used, IMHO, renaming it would be more clear.

Reply:
Change variable is a dangerous thing. The standard method are add an annotation, to indicate it.

Reply:
Did you mean the `deprecated` field option in protobuf?  It's written as `[deprecated = true]`, what you wrote is a c++ comment.

Reply:
you can use [deprecated = true]

Reply:
done